src/transaction_builder.ts

```diff
@@ -338,7 +335,15 @@ export default class TransactionBuilder {
    * Generate the payload for the previous signature by encoding address,  type and data
    */
   previousSignaturePayload() {
-    const bufCodeSize = intToUint32Array(this.data.code.length);
+    let bufContract: Uint8Array = intToUint32Array(0)
+    if (this.data.contract != undefined) {
+      const contract = this.data.contract
+      bufContract = concatUint8Arrays(
+        intToUint32Array(contract.bytecode.byteLength),
```

Comment:
quid if bytecode undefined ?

Reply:
Updated serialization, now there is a byte 1 if contract is filled 0 if not